lib/index.js

```diff
@@ -197,4 +197,9 @@ var reactA11y = (React, options = {}) => {
   };
 };
 
+reactA11y.restore = function(React) {
+  React.createElement = _createElement;
+  after.restorePatchedMethods();
```

Comment:
@ljharb Yes, `after.restorePatchedMethods()` restores every lifecycle method that was altered by react-a11y. The idea being that that called `a11y.restore(React)` takes care of cleaning up everything that was modified during the course of testing your components for issues.